src/theme/MainnetCountdown/useTimeLeft.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+
+// April 20th, 2023 @ 9am Pacific Time
+const LAUNCH_DATE = new Date("2023-04-20T16:00:00.000Z");
+
+function getTimeLeft() {
+  const now = new Date();
+  const diff = LAUNCH_DATE.getTime() - now.getTime();
```

Comment:
Might be a good idea to prevent this from going negative, and possibly do something different when it hits 0, like disable the modal

Reply:
Lol, yeah, good catch.